website/other/bezier-rs-demos/src/main.ts

```diff
@@ -77,3 +77,10 @@ if (splitPathName[1] === "bezier" && splitPathName[2] in bezierFeatures) {
 	(Object.keys(bezierFeatures) as BezierFeatureKey[]).forEach((feature) => renderBezierPane(feature, bezierDemos));
 	(Object.keys(subpathFeatures) as SubpathFeatureKey[]).forEach((feature) => renderSubpathPane(feature, subpathDemos));
 }
+
+// Scroll to specified hash if it exists
+const hash = window.location.hash;
+const target = document.querySelector(hash);
+if (target) {
+	target.scrollIntoView();
+}
```

Comment:
The browser already does this for you. But we want to hide all the others, not scroll to the specific one, if a valid hash fragment is present. And that should be achieved with CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/:target

Reply:
Disregard the idea about using CSS for performance concerns.

Reply:
The browser doesn't seem to be doing the scroll for me when going to a url with the hash directly and including this if statement solves it for me. I'm guessing that this might be because the examples are conditionally render.

Reply:
Try copying the URL, closing the tab, then opening a new tab and pasting the URL. I believe browsers don't re-scroll each time you hit enter in the URL bar or reload the page, it only scrolls when the hash changes or the page first loads.

Reply:
It wasn't working when the page first loads for me.

Reply:
Likely necessary because those elements aren't in the DOM yet.